prog/dftb+/prg_dftb/main.F90

```diff
@@ -236,29 +258,64 @@ subroutine runDftbPlus(env)
       end if
 
       call resetExternalPotentials(potential)
+
+      if (tReadShift) then
+        call uploadShiftPerL(fShifts, orb, nAtom, nSpin, potential%extShell)
+      end if
+
+
       call setUpExternalElectricField(tEField, tTDEField, tPeriodic, EFieldStrength,&
           & EFieldVector, EFieldOmega, EFieldPhase, neighborList, nNeighbor, iCellVec,&
           & img2CentCell, cellVec, deltaT, iGeoStep, coord0Fold, coord, EField,&
           & potential%extAtom(:,1), absEField)
-    
+
+
       call mergeExternalPotentials(orb, species, potential)
+        
+      ! Dirty trick to jump out in case of non-scc calculations with transport only
```

Comment:
Can't this set maxSccIter in the same way the regular non-SCC calculations deal with this?